packages/fxa-auth-server/lib/senders/emails/partials/cadReminder/index.ts

```diff
@@ -8,13 +8,13 @@ export const cadReminder = `
   <mj-include path="./lib/senders/emails/css/cadReminder/index.css" type="css" css-inline="inline" />
   <mj-section>
     <mj-column>
-      <mj-text css-class="header-text"><span data-l10n-id="<%- templateName %>-title"><%- headerText %></span></mj-text>
-      <mj-image css-class="sync-logo" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
+      <mj-text css-class="text-xlarge mb-5"><span data-l10n-id="<%- templateName %>-title"><%- headerText %></span></mj-text>
```

Comment:
Hmm yeah see my concern here is someone might just use `text-xlarge` and then like, add `mb-3` or `mb-4` here or on `sync-logo` or on anything else here and then lose that consistency we'd gain from it being in a single class.